docs/read_the_docs/source/tutorial/architecture.rst

```diff
@@ -62,6 +62,7 @@ The BuildExistingModel and ApplyUpgrade meta measures call the following model m
   1     ResStockArguments             Model              No                      ResStock
   2     BuildResidentialHPXML         Model              No                      OS-HPXML
   3     BuildResidentialScheduleFile  Model              No                      OS-HPXML
+  4     ResStockArgumentsPostHPXML    Model              No                      ResStock
```

Comment:
I think the workflow makes sense. This allows for ResStock custom edits to the OS-HPXML workflow.